### DISCUSSION TITLE
Who is the halfrussian on GitHub?

### DISCUSSION THREAD

**Discussion by halfrussian** [community/community#121732] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Can someone answer, who is the halfrussian on GitHub?

**Comment by carmenmen** [comment, marked as answer]
You are the halfrussian

**Reply by halfrussian** [reply]
You are correct, thank you. I am the halfrussian on github

**Comment by ebndev** [comment]
Hi @halfrussian,

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community.

Thanks for understanding!
